osu.Game/Localisation/ToastStrings.cs

```diff
@@ -34,6 +34,26 @@ public static class ToastStrings
         /// </summary>
         public static LocalisableString RestartTrack => new TranslatableString(getKey(@"restart_track"), @"Restart track");
 
+        /// <summary>
+        /// "Beatmap Editor"
+        /// </summary>r
+        public static LocalisableString BeatmapEditor => new TranslatableString(getKey(@"beatmap_editor"), @"Beatmap Editor");
+
+        /// <summary>
+        /// "Beatmap Saved"
+        /// </summary>
+        public static LocalisableString EditorSaveBeatmap => new TranslatableString(getKey(@"beatmap_editor_save"), @"Beatmap Saved");
```

Comment:
I have no idea what's being done here.

Are you manually making these files? We generally use a rider tool to do this.

If you are manually doing it, you will need to pay very close attention to the format. The property name should match the key should match the string.

In this case, the string should be `Beatmap saved` (not title case) and the key should be `BeatmapSaved` / `beatmap_saved` to match.

Same applies to others.